app/models/regex.rb

```diff
@@ -37,7 +37,7 @@ def save
   end
 
   def destroy
-    redis.hdel(regex_key, pattern)
+    redis.del(regex_key, pattern)
```

Comment:
Why did you make this change?

Reply:
the regex page was not deleting the selected rows when the `remove` button was clicked. I assumed this was a typo because when I removed the "h" the delete feature began working again.